src/AutoRest.CSharp/Common/Output/Expressions/ValueExpressions/FormattableStringExpression.cs

```diff
@@ -15,18 +15,19 @@ namespace AutoRest.CSharp.Common.Output.Expressions.ValueExpressions
     /// </summary>
     internal sealed record FormattableStringExpression : ValueExpression
     {
-        public FormattableStringExpression(string format, IReadOnlyList<ValueExpression> args)
+        public FormattableStringExpression(string format, params ValueExpression[] args)
```

Comment:
shall we keep the `IReadOnlyList` overload?

Reply:
Sure.
I changed this because the `IReadOnlyList` overload is quite unconvenient. But we could keep it